c2corg_api/jobs/purge_expired_tokens.py

```diff
@@ -0,0 +1,22 @@
+from c2corg_api.models.token import Token
+from datetime import datetime
+from sqlalchemy.orm import sessionmaker
+
+import logging
+log = logging.getLogger(__name__)
+
+
+def purge_token():
+    now = datetime.utcnow()
+    session = sessionmaker()()
+
+    try:
+        query = session.query(Token).filter(Token.expire < now)
+        count = query.count()
+
+        log.info('Deleting %d expired token', count)
+        if count > 0:
+            query.delete(synchronize_session=False)
```

Comment:
`query.delete` returns the number of deleted rows, so you would not have to make a separate query:
http://docs.sqlalchemy.org/en/rel_1_0/orm/query.html#sqlalchemy.orm.query.Query.delete